lib/parallel_tests/cli.rb

```diff
@@ -142,7 +142,17 @@ def report_number_of_tests(groups)
       num_processes = groups.size
       num_tests = groups.map(&:size).inject(0, :+)
       tests_per_process = (num_processes == 0 ? 0 : num_tests / num_processes)
-      puts "#{num_processes} processes for #{num_tests} #{name}s, ~ #{tests_per_process} #{name}s per process"
+      puts "#{pluralize(num_processes, 'process'} for #{pluralize(num_tests, name)}, ~ #{pluralize(tests_per_process, name)} per process"
```

Comment:
```suggestion
      puts "#{pluralize(num_processes, 'process')} for #{pluralize(num_tests, name)}, ~ #{pluralize(tests_per_process, name)} per process"
```

Reply:
Whoops, you’re 100% correct! Sorry about that. Force-pushed!